### PR TITLE
Addressing Org Membership Inconsistency

### PR DESCRIPTION
**Pull request by safaci2000**

### CODE REVIEW

File: cli/backup/organizations.go [hunk 1]
```diff
@@ -53,26 +52,42 @@ func newOrganizationsListCmd() simplecobra.Commander {
...
 		},
 		RunFunc: func(ctx context.Context, cd *simplecobra.Commandeer, rootCmd *support.RootCommand, args []string) error {
 			filter := service.NewOrganizationFilter(parseOrganizationGlobalFlags(cd.CobraCommand)...)
-			slog.Info("Listing organizations for context", "context", config.Config().GetGDGConfig().GetContext())
-			rootCmd.TableObj.AppendHeader(table.Row{"id", "organization Name", "org slug ID", "HomeDashboardUID", "Theme", "WeekStart"})
-			listOrganizations := rootCmd.GrafanaSvc().ListOrganizations(filter)
+			includePreferences, _ := cd.CobraCommand.Flags().GetBool("with-preferences")
+
+			if includePreferences {
+				rootCmd.TableObj.AppendHeader(table.Row{"id", "organization Name", "org slug ID", "HomeDashboardUID", "Theme", "WeekStart"})
```
Comment (daldoyle) [inline cli/backup/organizations.go:62]: There are a few places like this that feel like there's copy/paste and chance for desynchronization. 

Could we do something like the following pseudo-code:

rowHeaders = ["id", "organization Name", "org slug ID"]
if includePreferences {
    rowHeaders.append(["HomeDashboardUID", "Theme", "WeekStart"])
}
rootCmd.TableObj.AppendHeader(table.Row{rowHeaders})

File: cli/backup/organizations.go [hunk 2]
```diff
@@ -53,26 +52,42 @@ func newOrganizationsListCmd() simplecobra.Commander {
...
 			} else {
 				for _, org := range listOrganizations {
-					rootCmd.TableObj.AppendRow(table.Row{org.Organization.ID,
-						org.Organization.Name,
-						slug.Make(org.Organization.Name),
-						org.Preferences.HomeDashboardUID,
-						org.Preferences.Theme,
-						org.Preferences.WeekStart,
-					})
+					if includePreferences {
```
Comment (daldoyle) [inline cli/backup/organizations.go:77]: Same as above re: copy/paste columns.

File: cli/backup/organizations.go (hunk 1 above)
Comment (safaci2000) [reply cli/backup/organizations.go]: Fixed

File: cli/backup/organizations.go (hunk 2 above)
Comment (safaci2000) [reply cli/backup/organizations.go:80]: Fixed

**Review by daldoyle — APPROVED** [review]
Looks great!

**Pull request merged**